### DISCUSSION TITLE
EmacsTalk 听友微信群

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/joinus#18] [Announcements]
![7366c51ad33743756c207471fdd3b1ef](https://github.com/user-attachments/assets/399dafd5-c37f-41f1-aaaa-9462fb3d4b73)

如二维码过期，请在此留言，我们会及时更新，谢谢！更多群聊方式见：https://emacs.liujiacai.net/

**Comment by himself67** [comment]
The ORCode has expired

**Reply by jiacai2050** [reply]
Updated

**Reply by fqmoon** [reply]
过期啦麻烦补个😉

**Reply by jiacai2050** [reply]
更新了

**Reply by nahso** [reply]
过期了麻烦补个（

**Reply by jiacai2050** [reply]
Updated
